nilearn/plotting/displays.py

```diff
@@ -721,7 +720,11 @@ def _show_colorbar(self, cmap, norm, threshold=None):
                          x_adjusted_width,
                          height - (self._colorbar_margin['top'] +
                                    self._colorbar_margin['bottom'])]
-        self._colorbar_ax = figure.add_axes(lt_wid_top_ht, axis_bgcolor='w')
+        self._colorbar_ax = figure.add_axes(lt_wid_top_ht)
+        if matplotlib.__version__ >= LooseVersion("1.4"):
```

Comment:
I think that the right way to do the comparison is LooseVersion(matplotlib.__version__) >= LooseVersion("1.4")

(same comment below)